packages/components/src/components/post-tooltip-trigger/post-tooltip-trigger.tsx

```diff
@@ -0,0 +1,70 @@
+import { Component, Element, Prop } from '@stencil/core';
+import { isFocusable } from '../../utils/is-focusable';
+import { version } from '@root/package.json';
+import 'long-press-event';
+
+@Component({
+  tag: 'post-tooltip-trigger',
+})
+export class PostTooltipTrigger {
+  /**
+   * Link the trigger to a tooltip with this id
+   */
+  @Prop() for: string;
+
+  @Element() host: HTMLPostTooltipTriggerElement;
+
+  private trigger: HTMLElement;
+  private localInterestHandler;
+  private localInterestLostHandler;
+
+  constructor() {
+    this.localInterestHandler = this.interestHandler.bind(this);
+    this.localInterestLostHandler = this.interestLostHandler.bind(this);
+  }
+
+  componentDidLoad() {
+    this.host.setAttribute('data-version', version);
+
+    if (this.host?.children.length > 0 && this.host.children[0].nodeType === 1) {
+      this.trigger = this.host.children[0] as HTMLElement;
+    } else {
+      this.trigger = this.host;
+    }
+
+    // Ensure trigger is focusable
+    if (!isFocusable(this.trigger)) {
+      this.trigger.setAttribute('tabindex', '0');
+    }
+
+    // Add tooltip to aria-describedby
+    const describedBy = this.trigger.getAttribute('aria-describedby');
+    if (!describedBy?.includes(this.for)) {
+      const newDescribedBy = describedBy ? `${describedBy} ${this.for}` : this.for;
+      this.trigger.setAttribute('aria-describedby', newDescribedBy);
+    }
+
+    this.host.addEventListener('pointerover', this.localInterestHandler);
```

Comment:
Why not adding the event listeners on the trigger instead of the host?

Reply:
Just in case the trigger gets replaced or re-rendered by react/angular in which case the event bindings would be lost. If the trigger is re-rendered, the lifecycle hooks will trigger again.